### PR TITLE
Adding Starbucks and ZEE5.

### LINKED ISSUE #152: Lead: Starbucks
### Summary

Saw this tweet from Google I/O:

<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">The <a href="https://twitter.com/Starbucks?ref_src=twsrc%5Etfw">@Starbucks</a> PWA resulted in 2x daily active users. We&#39;re seeing users order via the desktop at nearly the same rate of mobile browsers. <a href="https://twitter.com/hashtag/io18?src=hash&amp;ref_src=twsrc%5Etfw">#io18</a> <a href="https://t.co/qczusLBElF">pic.twitter.com/qczusLBElF</a></p>&mdash; David Brunelle @ Google I/O (@davidbrunelle) <a href="https://twitter.com/davidbrunelle/status/993960071406080000?ref_src=twsrc%5Etfw">May 8, 2018</a></blockquote>

I'd love to know if there's a better source for the stat, but thought I'd make an issue tracking it because Starbucks would make a great addition.

### Source

https://twitter.com/davidbrunelle/status/993960071406080000

### Tags

¯\\\_(ツ)_/¯

### PR DESCRIPTION
**Pull request by grigs**
## Overview

Adds Starbucks and ZEE5. ZEE5 is only available in India, but it is a pretty cool PWA.

Fixes #152 

I decided to link to the tweet instead of the video:
https://youtu.be/QCoQs9NO9q0?t=1871

Because the tweet contained a little more detail about how the PWA was seeing traffic on desktop.

## Screenshots

Screen Shot 2020-02-14 at 4 55 06 PM

---

/CC @cloudfour/pwastats

### CODE REVIEW

**Comment by grigs** [comment]
That's weird. Not sure what I did wrong with my commit. I tried to commit a bunch of files at once so I probably messed it up that way.

I don't know how to remove the `Gemfile.lock`.

File: _posts/2020-02-14-starbucks.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Starbucks
+sourceURL: https://twitter.com/davidbrunelle/status/993960071406080000?ref_src=twsrc%5Etfw
+
```
Comment (tylersticka) [inline _posts/2020-02-14-starbucks.md]: Nit: Unnecessary line break

File: _posts/2020-02-14-starbucks.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+tags:
+  - Performance
+  - E-Commerce
+  - Service Worker
+  - Conversions
```
Comment (tylersticka) [inline _posts/2020-02-14-starbucks.md]: Nit: Tags are not alphabetized.

File: _posts/2020-02-14-zee5.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: ZEE5
+sourceURL: https://www.bgr.in/news/zee5-launches-a-new-progressive-web-app-in-an-effort-increase-its-reach-866037/
+tags:
+  - Performance
+  - E-Commerce
+  - Service Worker
+  - Media
```
Comment (tylersticka) [inline _posts/2020-02-14-zee5.md]: Nit: Tags are not alphabetized.

**Comment by grigs** [comment]
@tylersticka I still see the lock file changes in the file section. Did I somehow do it again?

**Comment by tylersticka** [comment]
@grigs No, but for whatever reason I cannot for the life of me get it to revert both files. The changes in this version are to some dependencies, they are bumping them _forward_, and they un-break Netlify, so I stopped trying.

If that's concerning, you'll need to ask a _real_ developer to help. I'm stumped.

**Comment by grigs** [comment]
If I understand what you mean by "unbreak," it sounds like I could merge the PR because the Netlify deploy is working, correct?

**Comment by tylersticka** [comment]
@grigs Yep, merge away!

**Pull request merged**